### PR TITLE
9224 - revert workflow metadata block in Solr schema

### LINKED ISSUE #9224: Revert adding workflow metadata block field to Solr Schema
As discussed for #7844 / #7877, we shall not ship a `schema.xml` with experimental metadata block fields in it.

Reverting the changes, add a few lines to docs.

### PR DESCRIPTION
**Pull request by poikilotherm**
**What this PR does / why we need it**:

As discussed for #7877, we don't want experimental metadata block fields in the out-of-the-box `schema.xml`, but introduced them for the workflow schema in #8812.

**Which issue(s) this PR closes**:

Closes #9224

**Special notes for your reviewer**:
Please review the docs changes to give people a heads up.

See https://dataverse-guide--9225.org.readthedocs.build/en/9225/user/appendix.html#experimental-metadata for preview.

**Suggestions on how to test this**:
Delete the fields, reindex, you're done.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:
Nope.

**Is there a release notes update needed for this change?**:
:battery: included

**Additional documentation**:
:battery: included

### CODE REVIEW

File: doc/sphinx-guides/source/user/appendix.rst
Comment (pdurbin) [inline doc/sphinx-guides/source/user/appendix.rst:41]:
```suggestion
Please note: these custom metadata schemas are not included in the Solr schema for indexing by default, you will need
```

**Comment by pdurbin** [comment]
Looks great. I made a tiny suggestion. Otherwise this is ready for QA. Gave it a 3 (the smallest possible) as a size.

**Comment by poikilotherm** [comment]
> Looks great. I made a tiny suggestion. Otherwise this is ready for QA. Gave it a 3 (the smallest possible) as a size.

@pdurbin I think you need to approve the change or @kcondon will not be able to merge.

**Comment by pdurbin** [comment]
@poikilotherm right, merging is blocked until someone clicks "approve" in a review.

But first things first. This PR hasn't been prioritized yet. It's currently at the bottom of the HERMES column at https://github.com/orgs/IQSS/projects/34/views/1 . The goal is the "1️⃣ ▶ORDERED BACKLOG (Stefano)" column and eventually into a sprint at https://github.com/orgs/IQSS/projects/2 . I hope that makes sense. I think we should document this as part of this issue:

- https://github.com/IQSS/dataverse/issues/9157

**Comment by mreekie** [comment]
backlog grooming
- @poikilotherm I got here by way of #7877
- I just want to reaffirm what Phil said about the priority. When Stefano chooses what to pull into an upcoming sprint, he looks for the items at the top of any column first.  
- I saw today that this is at the bottom of your list and figured I'd ping to double check
- If it's your intent that this be at the bottom of your priority then no action is needed.

**Review by scolapasta — APPROVED** [review]

**Review by pdurbin — APPROVED** [review]
These changes look good to me.

**Comment by pdurbin** [comment]
Related:

- #9390

**Pull request merged**
